src/Redux/Books/Books.js

```diff
@@ -1,23 +1,27 @@
 const ADD_BOOK = 'bookstore/books/addBook';
 const REMOVE_BOOK = 'bookstore/books/removeBook';
-const defaultData = [
-  { id: 1, title: 'A book to you', author: 'Esmatullah' },
-  { id: 2, title: 'A book of better life', author: 'Ahmadzai' },
-];
-export default function removeReducer(state = defaultData, action) {
+const FETCH_DATA = 'bookstore/books/fetchData';
+export default function addRemoveReducer(state = [], action) {
   switch (action.type) {
+    case FETCH_DATA:
+      return [...action.payload];
     case ADD_BOOK:
-      return [...state, action.books];
+      return [...state, action.payload];
     case REMOVE_BOOK:
```

Comment:
Removing a book functionality is not working. It removes the book from the redux state but does not remove it from the server. Kindly fix this issue.